contribs/gnodev/cmd/gnodev/setup_web.go

```diff
@@ -12,7 +12,11 @@ import (
 func setupGnoWebServer(logger *slog.Logger, cfg *devCfg, dnode *gnodev.Node) http.Handler {
 	webConfig := gnoweb.NewDefaultConfig()
 	webConfig.RemoteAddr = dnode.GetRemoteAddress()
-	webConfig.HelpRemote = dnode.GetRemoteAddress()
+	if cfg.webHelpRemote != "" {
+		webConfig.HelpRemote = cfg.webHelpRemote
+	} else {
+		webConfig.HelpRemote = dnode.GetRemoteAddress()
+	}
```

Comment:
Initialize `webConfig` on the main struct and manage all flag updates using flags. In essence, inherit and share flag logic from subcomponents like gnoweb, gnoland, etc.

Reply:
Now initializing in main `webConfig: gnoweb.NewDefaultConfig(),`